keyboards/handwired/illera16/info.json

```
@@ -0,0 +1,66 @@
+{
+    "manufacturer": "victor_illera",
+    "keyboard_name": "illera16",
+    "maintainer": "villeramassana",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "rgb_matrix": false,
+        "encoder": true,
+        "encoder_map": true
```

Its been mentioned a few times now, the encoder map feature **cannot** be enabled at the keyboard level. The correct location would be in a `keyboards/handwired/illera16/keymaps/default/rules.mk` file.

```suggestion
        "encoder": true
```